gammapy/utils/energy.py

```diff
@@ -237,7 +237,7 @@ def from_lower_and_upper_bounds(cls, lower, upper, unit=None):
         """
 
         # np.append renders Quantities dimensionless
-        # http://astropy.readthedocs.org/en/latest/known_issues.html#quantity-issues
+        # http://docs.readthedocs.org/en/latest/known_issues.html#quantity-issues
```

Comment:
I think this one should point here:
http://docs.astropy.org/en/latest/known_issues.html#quantity-issues